docs/software/social.md

```diff
@@ -285,4 +286,49 @@ Official documentation: <https://matrix.org/docs/guides>
 Source code: <https://github.com/matrix-org/synapse>  
 License: [Apache 2.0](https://github.com/matrix-org/synapse/blob/develop/LICENSE)
 
+***
+
+## microblog.pub
+
+A self-hosted, single-user, ActivityPub powered microblog.
+
+![microblog.pub blog screenshot](../assets/images/dietpi-software-social-microblogpub1.jpg){: width="400" height="337" loading="lazy"}
+
+=== "Installation / configuration"
+
+    The software installs automatically with a default config. Therefore it is necessary to reconfigure it using the `microblog-pub` script, which loads the environment, and brings up the app's own configuration wizard.  
+    This circumstance is signalled with the following dialog at the end of the installation process:
+
+    ![microblog.pub installation screenshot](../assets/images/dietpi-software-social-microblogpub2.jpg){: width="640" height="142" loading="lazy"}
+
+    To execute this reconfiguration, you can enter the following commands:
+
+    ```sh
+    bash
```

Comment:
This replaces the current shell with the new one, so there is no nested subshell 🙂:
```suggestion
    exec bash
```